config/iam/envoy-iam-policy.json

```diff
@@ -0,0 +1,20 @@
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": [
+                "appmesh:StreamAggregatedResources"
```

Comment:
Similar to above, this will need a preview variant (`appmesh-preview:StreamAggregatedResources`).

The managed policies for Envoy *only* include App Mesh actions (no ACM for security reasons), so I don't think the managed policy approach really helps here.